scripts/sel4.sh

```diff
@@ -55,16 +58,17 @@ as_root apt-get install -y --no-install-recommends \
     u-boot-tools \
     clang-11 \
     g++-10 \
-    g++-10-aarch64-linux-gnu \
     g++-10-arm-linux-gnueabi \
     g++-10-arm-linux-gnueabihf \
     gcc-10 \
-    gcc-10-aarch64-linux-gnu \
     gcc-10-arm-linux-gnueabi \
     gcc-10-arm-linux-gnueabihf \
     gcc-10-base \
-    gcc-10-multilib \
     gcc-riscv64-unknown-elf \
+    gcc-10-x86-64-linux-gnu:arm64 \
```

Comment:
The added lines are AArch64 specific and so we should have a list of the AArch64 deps and x86 cross-compiler deps and then just add all of the deps together before installing depending on the arch we're building the container for.

Reply:
Have now made these conditional.